### ISSUE TITLE
File permissions

### ISSUE THREAD

**Issue by pierreganty** [elenagutiv/Linearisation-2015#12]
Asking for executable files with the command `find . -type f -perm +111 -print` returns the following files which should not be executable:

```
...
./case-studies/P0/palindrome.nts.horn
./case-studies/P0/summ_ackermann.horn
./case-studies/P0/summ_b.horn
./case-studies/P0/summ_binsearch-inlined.horn
./case-studies/P0/summ_binsearch.horn
./case-studies/P0/summ_cil.casts.horn
./case-studies/P0/summ_evenodd.horn
./case-studies/P0/summ_formals.horn
./case-studies/P0/summ_g.horn
./case-studies/P0/summ_globals.horn
./case-studies/P0/summ_h.horn
./case-studies/P0/summ_local-ctx-call.horn
./case-studies/P0/summ_locals.horn
./case-studies/P0/summ_locals2.horn
./case-studies/P0/summ_locals3.horn
./case-studies/P0/summ_locals4.horn
./case-studies/P0/summ_mccarthy.horn
./case-studies/P0/summ_mccarthy2.horn
./case-studies/P0/summ_mccarthy3.horn
./case-studies/P0/summ_multi-call.horn
./case-studies/P0/summ_nested.horn
./case-studies/P0/summ_ptr_assign.horn
./case-studies/P0/summ_recursive.horn
./case-studies/P0/summ_rholocal.horn
./case-studies/P0/summ_rholocal2.horn
./case-studies/P0/summ_SG1.horn
./case-studies/P0/summ_SG1.r.horn
./case-studies/P0/summ_SG2.horn
./case-studies/P0/summ_SG2.r.horn
./case-studies/P0/summ_SG3.horn
./case-studies/P0/summ_slicing.horn
./case-studies/P0/summ_summs.horn
./case-studies/P0/summ_typedef.horn
./case-studies/P0/summ_x.horn
...
```

**Comment by elenagutiv** [comment]
Done!

**Comment by pierreganty** [comment]
Solved with cf2850d2fe48a0f7d6f9f58bc4fc5c3ad8dc1bbd
